### ISSUE TITLE
User Show Page

### ISSUE THREAD

**Issue by nolds9** [rubyforgood/share_christmas#59]
As a donor I would like to:
- [ ] see all my matches and information about each recipient
- [x] see instructions for my donation

**Comment by craig-riecke** [comment]
We have a start for this.  Right now, the screen only lists one match or one org_campaign (I forget).  The final version should list all matches for all in-progress org campaigns, and at the bottom a historical record of past campaigns.

**Comment by Josecc** [comment]
So far it lists all matches for one campaign I believe, so we need only work on showing past campaigns maybe in a dropdown. I think the focus should be on the current campaign so maybe have a section called "Past Campaigns" at the bottom and list them there, not grabbing a ton of attention.

**Comment by craig-riecke** [comment]
👍 Yeah, that sounds like a better approach.  Givers probably won't be interested in past campaigns.

**Comment by atzorvas** [comment]
Well the current code is more than hacky IMHO and we've introduced a bug with #107.

I would suggest cleaning up anything related to membership-recipients-campaigns into the existing implementation. 

We could also store the cleanup commit link inside the first post, so we can come back later and import any functionality/markup while creating the proper functionality.

**Comment by craig-riecke** [comment]
Yeah, that doesn't surprise me.  There are three constituencies with stories: donors, org admins, and volunteer center admins.  At RFG we focused on donors and volunteer center admins.  After RFG we switched our focus to org admins, changed the data models, and just haven't got back to refactoring the donor portion (which User Show is a part of) yet.  

Anyone is welcome to take a crack at fixing it!

**Comment by atzorvas** [comment]
@craig-riecke should we try to fix it or just drop it and re-implement in the future? I think the second is faster, since we want to get to the MVP asap and also we can have less headaches :smiley:

**Comment by craig-riecke** [comment]
@atzorvas The donor being able to see the match and donation instructions for the "current" campaign needs to be part of the MVP.   (Seeing past matches is not so important and can be dropped).  If that means rewriting this page and/or moving it, I'm fine with that.
